db/migrate_cards/data/metric_stuff_13.json

```diff
@@ -0,0 +1,121 @@
t\r\n\t<div class=\"row metric-info\">\r\n\t \r\n\t <!--Ratings and company list -->\r\n\r\n\t\t<div class=\"col-md-6 rate border-right\">\r\n\t\t  <div class=\"row\">{{+right sidebar}}</div>\r\n\t\t</div>\r\n\t\t\r\n\t\t\r\n\t\t<!--Rightside -->\r\n\t\t\r\n\t\t<div class=\"col-md-6 wiki border-left double-border-fix-left\">\r\n\t\t  \r\n\t\t  <!--tabs-->\r\n\r\n\t\t  <div class=\"tabbable\">\r\n\t\t\t\t<ul class=\"nav nav-tabs\">\r\n\t\t\t\t\t<li class=\"active\">\r\n\t\t\t\t\t\t<a href=\"#metric-details\" data-toggle=\"tab\"><h4>Details</h4></a>\r\n\t\t\t\t\t</li>\r\n\t\t\t\t\t<li>\r\n\t\t\t\t\t\t<a href=\"#metric-references\" data-toggle=\"tab\"><h4><i class=\"fa fa-globe\"></i> Sources</h4></a>\r\n\t\t\t\t\t</li>\r\n\t\t\t\t\t<li>\r\n\t\t\t\t\t\t<a href=\"#metric-discussion\" data-toggle=\"tab\"><h4><i class=\"fa fa-comment\"></i> Discussion</h4></a>\r\n\t\t\t\t\t</li>\r\n\t\t\t\t</ul>\r\n\t\t\t\t\r\n\t\t\t\t\r\n\t\t\t\t<div class=\"tab-content\">\r\n    \t\t\t\t\t<div class=\"tab-pane active\" id=\"metric-details\">\r\n    \t\t\t\t\t  \r\n                <div class=\"row clearfix\">\r\n                      <div class=\"col-md-6 padding-top-10\">\r\n                        <div class=\"btn-group\" role=\"group\" >\r\n                          <a class=\"btn btn-primary slotter\"\r\n                             href=\"/new/metric_value?slot[metric]={{_|cgi_escape_name}}\">\r\n                            <span class=\"fa fa-plus\"></span>\r\n                            Add new value \r\n                          </a>\r\n                          <a class=\"btn btn-default slotter\" href='/{{How to research and add Metric Values|linkname}}?layout=modal' data-toggle='modal' data-target='#modal-main-slot'>\r\n                             <small>Help</small>\r\n                          </a>\r\n                        </div> \r\n                      </div>\r\n                </div>  \r\n                <div class=\"row\">\r\n\r\n                  <div class=\"row-data\">\r\n                      {{+About|titled_with_edits;title:About;show:menu}}\r\n                  </div>\r\n     \r\n                  <div class=\"row-data\">\r\n                    {{+Methodology|titled_with_edits;title:Methodology;show:menu}}\r\n                  </div>   \r\n                </div> \r\n                <div class=\"row\">\r\n        \t\t\t\t\t    <div class=\"row-data titled-view\">\r\n        \t\t\t\t\t     <div class=\"card-header\">Value Type</div>\r\n                         <div class=\"col-md-11 nopadding pull-right\">\r\n                \t\t\t\t  <h5 class=\"pull-left\">Unit:</h5> <h5 class=\"pull-left\">{{+Unit|content}}</h5>\r\n                \t\t\t\t</div>\r\n                \t\t\t\t\r\n                \t\t\t\t<div class=\"col-md-11 nopadding  pull-right\">\r\n                \t\t\t\t  <h5 class=\"pull-left\">Range:</h5> <h5 class=\"pull-left\">{{+Range|content}}</h5>\r\n                \t\t\t\t</div>\r\n                      </div>\r\n              \
```

Comment:
we have to solve this load bug. Repeating the hard-coded coffeescript in a card is not a reasonable solution